src/librustdoc/clean/utils.rs

```diff
@@ -493,8 +493,8 @@ pub fn print_evaluated_const(cx: &DocContext<'_>, def_id: DefId) -> Option<Strin
             (_, &ty::Ref(..)) => None,
             (ConstValue::Scalar(_), &ty::Adt(_, _)) => None,
             (ConstValue::Scalar(_), _) => {
-                let const_ = ty::Const { val: ty::ConstKind::Value(val), ty };
```

Comment:
Here you can keep the original, as the value isbnot required to be interned

Reply:
I think it actually needs to be interned to be printed, I get the 'could not lift for printing' otherwise.

Reply:
Ah indeed